jupyterlab/commands.py

```diff
@@ -2230,10 +2234,13 @@ def noop(x, y, z):
     return return_value
 
 
-def _is_disabled(name, disabled=[]):
+def _is_disabled(name, disabled={}):
     """Test whether the package is disabled.
     """
-    for pattern in disabled:
+    for pattern, value in disabled.items():
+        # skip packages explicitely marked as disabled
```

Comment:
Should it be _not_ disabled?
```suggestion
        # skip packages explicitly marked as not disabled
```

Reply:
Yes, thanks for catching this!